calyx-py/calyx/queue_data_gen.py

```diff
@@ -70,20 +70,16 @@ def dump_json(num_cmds, no_err: bool, queue_size: Optional[int]=None, nwc=False,
     The data itself is populated randomly, following certain rules:
     - It has three "memories": `commands`, `values`, and `ans_mem`.
     - Optional memories `ranks` and `times` are included for queues primed for non-work-conserving algorithms.
-    - The `commands` memory has `num_cmds` items, which range from 0-2 for work-conserving policies,
-        and from 0-4 for non-work-conserving. They are as follows:
+    - The `commands` memory has `num_cmds` items, which range from 0-1 for work-conserving policies,
+        and from 0-1 for non-work-conserving. They are as follows:
```

Comment:
Maybe just tidy this up a little... it's 0 or 1 regardless of whether it's WC or NWC, right?